analyses/pedcbio-sample-name/README.md

```diff
@@ -26,23 +27,24 @@ For TARGET, TCGA, and GTEx samples, `Kids_First_Participant_ID` column is used a
   - For TCGA, `Kids_First_Participant_ID` + `sample_id` + `aliquot_id` is used as formatted sample ID
   - For GTEx, `Kids_First_Participant_ID` + `aliquot_id` is used as formatted sample ID
 
+For DGD, use `aliquot_id`, but remove test name
+
 Usage:
-  ```
-Rscript -e "rmarkdown::render('pedcbio_sample_name_col.Rmd', clean = TRUE)"
+```sh
+# In repo:
+Rscript --vanilla pedcbio-sample-name/pedcbio_sample_name_col.R -h path-to-histolgies-file.tsv"
```

Comment:
Assuming you're running from withing the folder, I would suggest removing the folder name. Also the option is `-i`, not `-h`

`Rscript --vanilla pedcbio_sample_name_col.R -i "path-to-histolgies-file.tsv"`

Reply:
@migbro can you also write out the path here and even though this is just a single r script, can you create the shell to run it to be consistent with other modules? Noticed it was deleted. Thanks!

Reply:
I can try...I could not for the life of me find the histologies file in this repo...and even if I did, for this to work, I'd either have to add a "ignore methylation" or some kind of ignore datatype filter for it to work. Perhaps that latter feature might be worth it to sort of future-proof if we get a new data type that isn't loaded into cBio.

Reply:
@kelseykeith as for finding the `-WXS` extensions, definitely desired for RNA seq when a matched DNA samples exists. The reason being is sometimes a WGS and WXS sample exists, and we need the `-WXS` to differentiate...and if the RNA sample is associated with that WXS, then the sample name covering the situation will still have that suffix. Basically, cBio sample names need to tie DNA/RNA biospecimens from the same location, same biopsy/collection event. Because of the situation outlined, it may cause a sample name with an DNA-associated assay type to be associated with an RNA sample.

Reply:
What histologies file are you using now?

Reply:
Well, the pre-release one you sent me. When the official one is ready, I expect to pull it from here:
`s3://d3b-openaccess-us-east-1-prd-pbta/open-targets/v12/histologies.tsv`, which I can't use as the `-i` input, since it's an aws object. I could try for "in-repo-mode" to say the use must put that file in `inputs` dir, then modify Run's loop to only look for csv to not accidentally start parsing the hist file when parsing the cBio ID files. Then a shell script would work

Reply:
Ahh I see - so it will be in the `/data` directory post release, but perhaps you can stack this PR onto https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/336, in which case it resides in https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/dev/analyses/molecular-subtyping-integrate/results/histologies.tsv